source/pdo_sqlsrv/config.m4

```diff
@@ -52,8 +52,14 @@ if test "$PHP_PDO_SQLSRV" != "no"; then
   fi
     AC_MSG_RESULT($pdo_sqlsrv_inc_path)
     
-           
+  HOST_OS_ARCH=`uname`         
+  if test "${HOST_OS_ARCH}" = "Darwin"; then
+      MACOSX_DEPLOYMENT_TARGET=`sw_vers -productVersion` 
+  fi

```

Comment:
It should be inside the if block.
Also, the previous way of checking the OS was wrong.